boards/lilygo/ttgo_t7v1_5/doc/index.rst

```diff
@@ -0,0 +1,231 @@
+.. _ttgo_t7v1_5:
+
+Lilygo TTGO T7 V1.5
+#################
+
+Overview
+********
+
+LILYGO® TTGO T7 Mini32 V1.5 ia an IoT mini development board
+based on the Espressif ESP32-WROVER-E module.
+
+It features the following integrated components:
+- ESP32 chip (240MHz dual core, 520KB SRAM, Wi-Fi, Bluetooth)
+- on board antenna
+- Micro-USB connector for power and communication
+- JST GH 2-pin battery connector
+- LED
+
+.. figure:: img/ttgo_t7v1_5.webp
```

Comment:
You can remove thiswhoe figure entry entry here (documentation will automatically add it). Please, decrease image size as well.

Reply:
done